### PR TITLE
Feat/create more space

### PR DESCRIPTION
**Pull request by MatthijsSmets**
Select all and deselect all buttons have been replaced with a checkbox in the table header.

Padding on table rows have been minimized to create more space.
Metadata table visible under the editor has been moved above it and can be shown or hidden with a button.
Table header names and column values have been shortened, hovering on the table header will still show the full name.

Table row spacing can be altered in settings, font size is also changed with the spacing.

### CODE REVIEW

File: angular.json
Comment (Matthbo) [inline angular.json:1]: Im very curious what `buildTarget` is and why you changed to that instead of `browserTarget`. Ive never seen this before

Comment (MatthijsSmets) [reply angular.json:1]: When running ng serve this is the warning i get:
image

Comment (Matthbo) [reply angular.json:1]: Eh
image

**Comment by Matthbo** [comment]
Currently spacing 0 is the default but I would change that to 1 as the default. 
If 1 is the default then people that want it small can make it smaller but it generally has better accessibility for people with to click on the right item

**Review by Matthbo — APPROVED** [review]

**Pull request merged**